### PR TITLE
ACS-3225 Get rules REST tests.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package org.alfresco.rest.rules;
```
Comment (montgolfiere) [inline packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java:1]: Add header ?

**Review by montgolfiere — APPROVED** [review]
Nice to see TDD (eg. some -ve & empty list tests first). 

Looking forward to seeing more tests once we have the ability to create rules, inherit rules, link rules etc.

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java (hunk 1 above)
Comment (tpage-alfresco) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java:1]: Oh - that's interesting. We don't seem to be checking for headers on test files.  I'll add one anyway.

**Review by mpichura — APPROVED** [review]

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java (hunk 1 above)
Comment (tpage-alfresco) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/GetRulesTests.java:1]: I forgot to do this before merging, so I added it in a separate commit.

**Pull request merged**